stack.yaml

```diff
@@ -23,3 +26,6 @@ flags:
     lib-only: true
   these:
     assoc: false
+system-ghc: true
+compiler: ghc-9.2.3
+compiler-check: match-exact
```

Comment:
I'm not familiar with `stack.yaml` in general. Why are these needed?

Reply:
oh yes that accidenily slipped into the commit

Reply:
Fixed